### PR TITLE
Fix the types for default request options.

### PR DESCRIPTION
**Pull request by ruiaraujo**
One could not specify the port or protocol without using ```as any```

### CODE REVIEW

**Comment by ruiaraujo** [comment]
👍

File: lib/client.ts [hunk 1]
```diff
@@ -112,8 +112,8 @@ class ServiceClientStrictOptions {
 
     this.defaultRequestOptions = Object.assign({
       protocol: 'https:',
-      pathname: '/',
-      timeout: 2000
+      port: 443,
```
Comment (shuhei) [inline lib/client.ts]: This will be a breaking change for `{ protocol: 'http:' }` without an explicit port.

File: lib/client.ts [hunk 2]
```diff
@@ -112,8 +112,8 @@ class ServiceClientStrictOptions {
 
     this.defaultRequestOptions = Object.assign({
       protocol: 'https:',
-      pathname: '/',
-      timeout: 2000
```
Comment (shuhei) [inline lib/client.ts]: Why removing timeout?

Comment (ruiaraujo) [reply lib/client.ts]: general stupidity from early morning, hence the wip label 😄

**Comment by ruiaraujo** [comment]
Replaced by https://github.com/zalando-incubator/perron/pull/55

**Pull request closed without merging**